app/models/conference.rb

```diff
@@ -57,6 +57,9 @@ class Conference < ActiveRecord::Base
   validates_format_of :short_title, with: /\A[a-zA-Z0-9_-]*\z/
   validates :registration_limit, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
 
+  validates :payment_method, presence: true, null: false, inclusion: {in: %w(online offline),
```

Comment:
This type of data is called [enum](http://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html)